### PR TITLE
fix glitch with devil

### PR DESCRIPTION
**Pull request by vvvyyynet**
as discussed

### CODE REVIEW

File: src/routes/Devilstable.svelte [hunk 1]
```diff
@@ -91,14 +92,14 @@
 </script>
 
 <Brush
-	height={mobile ? brushDimension : height}
```
Comment (flicksolutions) [inline src/routes/Devilstable.svelte:94]: Why did you change the order? does that matter?

**Review by flicksolutions — APPROVED** [review]
thank you, it works fine. Just have one question to clarify.

File: src/routes/Devilstable.svelte (hunk 1 above)
Comment (vvvyyynet) [reply src/routes/Devilstable.svelte:94]: for the sake of consistency
